<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Aravindh-PC" failures="2" tests="11" name="RetryMulitpleFailedTestCases" time="0.255" errors="0" timestamp="01 May 2019 02:35:10 GMT">
  <testcase classname="mytests.RetryAnalyzer" name="test1" time="0.023">
    <skipped/>
  </testcase> <!-- test1 -->
  <testcase classname="mytests.RetryAnalyzer" name="test1" time="0.009">
    <skipped/>
  </testcase> <!-- test1 -->
  <testcase classname="mytests.RetryAnalyzer" name="test1" time="0.009">
    <skipped/>
  </testcase> <!-- test1 -->
  <testcase classname="mytests.RetryAnalyzer" name="test1" time="0.003">
    <skipped/>
  </testcase> <!-- test1 -->
  <testcase classname="mytests.RetryAnalyzer" name="test1" time="0.003">
    <failure type="junit.framework.AssertionFailedError" message="expected:&amp;lt;true&amp;gt; but was:&amp;lt;false&amp;gt;">
      <![CDATA[junit.framework.AssertionFailedError: expected:<true> but was:<false>
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.failNotEquals(Assert.java:282)
at junit.framework.Assert.assertEquals(Assert.java:64)
at junit.framework.Assert.assertEquals(Assert.java:149)
at junit.framework.Assert.assertEquals(Assert.java:155)
at mytests.RetryAnalyzer.test1(RetryAnalyzer.java:11)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase classname="mytests.RetryAnalyzer" name="test2" time="0.003">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase classname="mytests.RetryAnalyzer" name="test2" time="0.008">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase classname="mytests.RetryAnalyzer" name="test2" time="0.003">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase classname="mytests.RetryAnalyzer" name="test2" time="0.012">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase classname="mytests.RetryAnalyzer" name="test2" time="0.003">
    <failure type="junit.framework.AssertionFailedError" message="expected:&amp;lt;true&amp;gt; but was:&amp;lt;false&amp;gt;">
      <![CDATA[junit.framework.AssertionFailedError: expected:<true> but was:<false>
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.failNotEquals(Assert.java:282)
at junit.framework.Assert.assertEquals(Assert.java:64)
at junit.framework.Assert.assertEquals(Assert.java:149)
at junit.framework.Assert.assertEquals(Assert.java:155)
at mytests.RetryAnalyzer.test2(RetryAnalyzer.java:16)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase classname="mytests.RetryAnalyzer" name="test3" time="0.003"/>
</testsuite> <!-- RetryMulitpleFailedTestCases -->
